### DISCUSSION TITLE
Welcome to Card-Game-Simulator Discussions!

### DISCUSSION THREAD

**Discussion by davidmfinol** [finol-digital/Card-Game-Simulator#36] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with CGS.

**Comment by Bendell90** [comment]
Is there going to be a way to upload your own custom cards and game rules?
I essentially am looking for a good place to test new card games 
If it is possible can it

+Place 4 different decks w separate cards to draw from
+Player to use card when they want, at the same time
+Place counters in order to note wins?

**Reply by Bendell90** [reply]
But I really shoulda started with excellent job so far this is a cool idea especially for mobile!

**Reply by davidmfinol** [reply]
Yep, it is currently possible to create custom cards, though I am still working on an automated upload process!

+ It is currently possible to place 3 different decks
+ Each player can move any card they want, simultaneously
+ Adding counters is something I am working on

Thanks for the feedback! Please do let me know if you see anything else that could be improved.

**Comment by Gidonamor** [comment]
How do I create my own cards? I can't find that functionality, only create custom decks with existing cards. Is there a way to create my own cards and upload them?

**Reply by davidmfinol** [reply]
Hi @Gidonamor , instructions for creating cards are on the wiki: https://github.com/finol-digital/Card-Game-Simulator/wiki/Crash-Course-into-Game-Development-with-CGS
Hope this helps!
